mne/event.py

```diff
@@ -957,7 +957,7 @@ def make_fixed_length_events(
     from .io.base import BaseRaw
 
     _validate_type(raw, BaseRaw, "raw")
-    _validate_type(id, int, "id")
+    _validate_type(id, "int", "id")
```

Comment:
Another version is `id = _ensure_int(...)`. It's slightly preferred because it always returns a python int but either way is fine here I think

Reply:
Should be correctly handled by numpy here `id * np.ones(n_events, dtype=int)` in this case